includes/QueryPrinterFactory.php

```diff
@@ -7,21 +7,21 @@
 /**
  * Factory for "result formats", ie classes implementing QueryResultPrinter.
  *
- * @since 1.9
+ * @since 2.5 (since 1.9, renamed in 2.5)
  *
  * @ingroup SMW
  *
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
  */
-final class FormatFactory {
+final class QueryPrinterFactory {
```

Comment:
Sounds good to me. Feel free to move it. To me the allowing of dependency injection is higher prio and I won't mix both changes in one PR.

I'm not up to speed with if this factory is being used outside of SMW. Perhaps that is not a good idea to begin with, as you can already register the query printers via the global vars. Or would there be a need for extensions to access instances of the printers?